src/tutorials/third-party.md

```diff
@@ -65,6 +65,7 @@ Framework  | Credit | Date added
 [**API Platform** with a **ReactJS** client](https://github.com/GuGuss/platformsh-api-platform-admin) admin |[@GuGuss](https://github.com/GuGuss)|May 2017
 [**Backdrop** example](https://github.com/gmoigneu/platformsh-example-backdrop)|[@gmoigneu](https://github.com/gmoigneu)|May 2017
 [**Headless Drupal 8 with Angular**](https://github.com/GuGuss/headless-drupal8-platformsh)|[@GuGuss](https://github.com/GuGuss)|May 2017
+[**Joomla** example](https://twitter.com/edgette22/status/921103567771131904)|[@gmoigneu](https://github.com/gmoigneu)|May 2017
```

Comment:
You might have posted a bad link by mistake. It lead to a Twitter post about KFC following the spice girls...

Reply:
lol. Whoops. I posted that in Slack.